kinfer/__init__.py

```diff
@@ -1,3 +1,11 @@
 __version__ = "0.0.4"
 
-from . import export, inference
+import sys
+from importlib.util import find_spec
+
+from kinfer.python import export, inference
+
+sys.modules["kinfer.export"] = export
+sys.modules["kinfer.inference"] = inference
```

Comment:
this is physically painful to me

we should just let people do `from kinfer import export, inference`

which we can accomplish by doing `from .python import export, inference` in this file